### PR TITLE
User logging

### PR DESCRIPTION
**Pull request by manavbagai**

### CODE REVIEW

**Review by zlmonroe — CHANGES_REQUESTED** [review]
No PRs direct to master

File: banking_system/banking_system/settings.py [hunk 1]
```diff
@@ -83,8 +84,8 @@
     'default': {
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'ss_project',
-        'USER': 'bank_admin',
-        'PASSWORD': 'Password#0',
+        'USER': 'root',
```
Comment (zlmonroe) [inline banking_system/banking_system/settings.py]: preferably don't change these as this is what the remote server on AWS is running with

Comment (zlmonroe) [reply banking_system/banking_system/settings.py]: As a note for why I didn't just leave the AWS running with the default settings, I figured that would be a security hole that was easily exploitable when the other teams get our code

File: banking_system/banking_system/settings.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
 DEBUG = True
```
Comment (zlmonroe) [inline banking_system/banking_system/settings.py]: Somehow admin users can't log in on the django page. Is it possible the logging is causing a caught error or something? I thought this was a server issue but it works on other branches.

**Review by zlmonroe — CHANGES_REQUESTED** [review]
Might be an issue with admin login. Setting password for mysql should not change either. I have disable root remote login for security reasons on mysql on AWS

File: banking_system/banking_system/settings.py (hunk 2 above)
Comment (ripu23) [reply banking_system/banking_system/settings.py]: Do you mean the Django Admin page?

File: banking_system/user_management/custom_backend.py [hunk 3]
```diff
@@ -3,10 +3,11 @@
 
 
 class CustomAuthBackend(ModelBackend):
-    def authenticate(self, request, username=None, password=None):
+    def authenticate(self, request, email=None, password=None):
```
Comment (ripu23) [inline banking_system/user_management/custom_backend.py:6]: Did you test this ? When I did this, the request never came here.

Comment (manavbagai) [reply banking_system/user_management/custom_backend.py:6]: request was already there. I just changed the parameter name from username to email. I have tested it, working fine.

File: banking_system/banking_system/settings.py (hunk 2 above)
Comment (zlmonroe) [reply banking_system/banking_system/settings.py]: Correct. Does this work for you?

Comment (zlmonroe) [reply banking_system/banking_system/settings.py]: This was an issue. It has been fixed in the last commit.

**Review by zlmonroe — DISMISSED** [review]
Everything there works. Perhaps could use invalid login attempt logs, but this can be merged.

**Review by zlmonroe — APPROVED** [review]
Logging works, including on EC2 cluster. Looks good.

**Pull request merged**